File: programmers/level1/k번째수.js
function solution(array, commands) {
    return commands.map(command => {
        const [
          startPosition,
          endPosition,
          position,
        ] = command;

        const sortedArray = array
            .filter((_, index) => index >= startPosition - 1 && index <= endPosition - 1)
            .sort((a,b) => a - b);

        return sortedArray[position - 1];
    });
}

File: programmers/level1/k번째수.go
import "sort"

func solution(array []int, commands [][]int) []int {
	var result []int
	for _, command := range commands {
		slice := append([]int{}, array[command[0]-1:command[1]]...)
		sort.Ints(slice)
		result = append(result, slice[command[2]-1])
	}
	return result
}